rosplane/scripts/rosplane_gcs_launch.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Function to print help instructions
+print_help() {
+    usage
+    echo
+    echo "Options:"
+    echo "  -h                Show this help message and exit"
+    echo "  -s                Run as simulation of ROSflight"
+    echo "  -t                Run tuning version of ROSflight"
+    echo "  -r                Run the script with simulated transmitter"
+    echo "  -a                Aircraft option to pass to launch files"
+    echo "  aircraft          The aricraft parameter to be used in launch files, default is anaconda"
+    echo "  -b                Start a ROS bag"
+    ehco "  bag_name          Name of bag (optional)"
+    echo "  -o                Run the tmux session online (on remote host)"
+    echo "  user@host         Username and address of remote"
+    echo "  -d                Indicate that you should run a docker container of the given name (uses compose, and compose file should be in workspce directory)"
+    echo "  container       Name of container"
+    echo "  path/to/workspace Path to workspace"
+}
+
+usage() {
+    echo "Usage: $0 [-h] [-s] [-t] [-r] [-a aircraft] [-b [bag_name]] [-o user@host] [-d docker_container_name] path/to/ROSflight/workspace"
+}
+
+sim=false
+tuning=false
+rc_sim=false
+aircraft='anaconda'
+online=false
+docker=false
+bag=false
+
+# Parse options using getopts
+while getopts ":hstra:b:o:d:" opt; do
+    case $opt in
+        h)
+            
+            print_help
+            exit 0
+            ;;
+        s)
+            sim=true
+            ;;
+        t)
+            tuning=true
+            ;;
+        r)
+            rc_sim=true
+            if ! $sim; then
+              echo "Cannot run simulated transitter, while not running a simulation."
+              exit 1 
+            fi
+            ;;
+        a)
+            aircraft=$OPTARG
+            ;;
+        b)
+            bag=true
+            bag_name=$OPTARG
+            ;;
+        o)
+            online=true
+            user_host=$OPTARG
+            if $sim; then
+              echo "Cannot run simulat on a remote."
+              exit 1 
+            fi
+            ;;
+        d)
+            docker=true
+            container=$OPTARG
+            if ! $online; then
+              echo "Script not configured for Docker not on remote."
+            fi
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            print_help
+            exit 1
+            ;;
+    esac
+done
+
+shift $((OPTIND -1))
+
+# Get the file path
+filepath=$1
+
+# Check if filepath is provided
+if [ -z "$filepath" ]; then
+    echo "Absolute/relative file path is required"
+    print_help
+fi
+
+# Create a new tmux session
+tmux new-session -d -s rosplane_sim_session
+
+tmux source-file ~/.tmux.conf
```

Comment:
What would happen if this file didn't exist?

Reply:
I think the command would fail but proceed. It would just mean your mouse doesn't work to click. But we should test it.

Reply:
Actually I am going to remove it for now.

Reply:
I left this as a commented line with a not to uncomment it if the user want to use their tmux config.